<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601647670744' stop='1601647672104'>
    <name>Home Page</name>
    <title>Home Page</title>
    <test-cases>
        <test-case start='1601647670744' status='broken' stop='1601647672103'>
            <name>It Launch domain url and verify title and the header of 3protofile text</name>
            <title>It Launch domain url and verify title and the header of 3protofile text</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-85.0.4183.121'/>
            </parameters>
            <steps>
                <step start='1601647670760' status='broken' stop='1601647672103'>
                    <name>POST /session/:sessionId/url</name>
                    <title>POST /session/:sessionId/url</title>
                    <attachments>
                        <attachment title='Request' source='bb93c16d-ea88-4422-8607-798465fd04c4-attachment.json' type='application/json' size='61'/>
                    </attachments>
                    <steps>
                        <step start='1601647671779' status='passed' stop='1601647672100'>
                            <name>GET /session/:sessionId/screenshot</name>
                            <title>GET /session/:sessionId/screenshot</title>
                            <attachments>
                                <attachment title='Screenshot' source='2bee2855-914f-4cbf-a3ba-16f988855775-attachment.png' type='image/png' size='22476'/>
                            </attachments>
                            <steps/>
                        </step>
                    </steps>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>unknown error: net::ERR_NAME_NOT_RESOLVED
                  (Session info: chrome=85.0.4183.121)</message>
                <stack-trace>unknown error: net::ERR_NAME_NOT_RESOLVED
                  (Session info: chrome=85.0.4183.121)
                    at getErrorFromResponseBody (C:\Users\hp\Pictures\Automation Assesment\node_modules\webdriver\build\utils.js:121:10)
                    at WebDriverRequest._request (C:\Users\hp\Pictures\Automation Assesment\node_modules\webdriver\build\request.js:149:56)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)
                    at async Browser.runCommandWithHooks (C:\Users\hp\Pictures\Automation Assesment\node_modules\@wdio\sync\build\wrapCommand.js:88:21)
                    at Browser.runCommandWithHooks (C:\Users\hp\Pictures\Automation Assesment\node_modules\@wdio\sync\build\wrapCommand.js:82:22)
                    at Browser.wrapCommandFn (C:\Users\hp\Pictures\Automation Assesment\node_modules\@wdio\sync\build\wrapCommand.js:59:40)
                    at homepages.open (C:\Users\hp\Pictures\Automation Assesment\node_modules\.bin\test\pageobjects\page.js:11:24)
                    at homepages.open (C:\Users\hp\Pictures\Automation Assesment\node_modules\.bin\test\pageobjects\home.js:28:18)
                    at Context.&lt;anonymous> (C:\Users\hp\Pictures\Automation Assesment\node_modules\.bin\test\specs\e2e.js:9:17)
                    at Context.executeSync (C:\Users\hp\Pictures\Automation Assesment\node_modules\@wdio\sync\build\index.js:56:18)
                    at C:\Users\hp\Pictures\Automation Assesment\node_modules\@wdio\sync\build\index.js:80:70</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>